doc/dev/osd_internals/snaps.rst

```diff
@@ -126,3 +126,84 @@ up to 8 prefixes need to be checked to determine all hobjects in a particular
 snap for a particular PG.  Upon split, the prefixes to check on the parent
 are adjusted such that only the objects remaining in the PG will be visible.
 The children will immediately have the correct mapping.
+
+clone_overlap
+-------------
+Each SnapSet attached to the head object contains the overlapping intervals
```

Comment:
The HEAD may be removed with snapshots remaining, in which case the snapset moves somewhere else.
```suggestion
Each SnapSet contains the overlapping intervals
```

Reply:
The head can't be removed if we still have clones, added this as well to the docs in a separate commit (a5579450853a588f17c1c405483a511d517a8abd)

Reply:
Ah, I forgot that Sage made that change!